build.sbt

```diff
@@ -113,7 +113,21 @@ lazy val grpc =
     .dependsOn(testkit % Test)
     .enablePlugins(AkkaGrpcPlugin)
     .settings(akkaGrpcCodeGeneratorSettings += "server_power_apis", IntegrationTest / fork := true)
-    //.settings(Scala3.settings) // FIXME add scala3 support once circular dependency is resolved
+    .settings(Scala3.settings)
+
+lazy val grpcTest =
+  Project(id = "akka-projection-grpc-test", base = file("akka-projection-grpc-test"))
```

Comment:
just to align with other naming this should be `akka-projection-grpc-tests`
e.g. akka-actor-tests, akka-actor-typed-tests

Reply:
Sounds good, however in this project is singular it seems:
```
Project(id = "akka-projection-core-test", base = file("akka-projection-core-test"))
```

Reply:
ok, keep it singular then

Reply:
Too late :) https://github.com/akka/akka-projection/pull/796

Reply:
Let me know if you think it should be re-renamed though.

Reply:
doesn't matter, fine in either way